scipy/interpolate/tests/test_interpnd.py

```diff
@@ -216,6 +216,28 @@ def do_interp(interpolator, slice_rows, slice_cols):
         worker_thread_3.join()
         worker_thread_4.join()
 
+    @pytest.mark.parametrize("simplex_tolerance", [1, 10])
+    def test_interp_from_boundary(self, simplex_tolerance):
+         # regression test for https://github.com/scipy/scipy/issues/21910
+        coords = np.block(
+            [[0, np.arange(400, 500), np.zeros(100), 800],
+             [0, np.zeros(100), np.arange(400, 500), 800]]
+        )
+        values = np.ones(202)
+        interpolator = interpnd.LinearNDInterpolator(coords.T, values)
```

Comment:
A very minor ask if you're updating the PR (might do as a follow-up as well): would be nice to `@pytest.parametrize("interpolator_cls", [LinearNDInterpolator, CloughTocher2DInterpolator])` the test, too.

Reply:
Should I try combining the other tests for LinearNDInterpolator and CloughTocher2DInterpolator as well?  There's a few specific to one or the other, but several would seem to apply to both.